core_helpers/logging_objects.py

```diff
@@ -17,20 +17,20 @@ def write_test_summary(self):
         self.write('Total number of checks passed=%d\n----------------------\n************************\n\n'%self.pass_counter)
         self.write('Total number of mini-checks=%d'%self.mini_check_counter)
         self.write('Total number of mini-checks passed=%d'%self.mini_check_pass_counter)
+        self.make_gif()
+        if self.gif_file_name is not None:
+            self.write("Screenshots & GIF created at %s"%self.screenshot_dir)
+            self.write('************************')
         failure_message_list = self.get_failure_message_list()
         if len(failure_message_list) > 0:
-            self.write('\n--------FAILURE SUMMARY--------\n')
+            self.write('\n--------FAILURE SUMMARY--------\n',level="error")
             for msg in failure_message_list:
-                self.write(msg)
+                self.write(msg,level="error")
         if len(self.exceptions) > 0:
             self.exceptions = list(set(self.exceptions))
-            self.write('\n--------USEFUL EXCEPTION--------\n')
+            self.write('\n--------USEFUL EXCEPTION--------\n',level="error")
```

Comment:
Can you we set the level to `critical` for `Exceptions`?

Reply:
yes. I made changes.